test/support/annotations/annotation_writer.rb

```diff
@@ -0,0 +1,36 @@
+module Annotations
+  class AnnotationWriter
+    def header_template
+      File.read(File.dirname(__FILE__) + '/templates/header.erb')
+    end
+
+    def annotation_template
+      File.read(File.dirname(__FILE__) + '/templates/annotation.erb')
+    end
+
+    def initialize(filename)
+      @file = File.open(filename, 'w')
+      @file.write(header_template)
+    end
+
+    def close
+      @file.close
+    end
```

Comment:
If you use a block with the `File.open` method you don't have to remember anything :)